tests/test_prediction.py

```diff
@@ -50,7 +50,7 @@ def test_repr(test_object: any):
 def model_run(CLIENT):
     ds = CLIENT.create_dataset(TEST_DATASET_NAME)
     ds_items = []
-    for url in TEST_IMG_URLS:
+    for url in TEST_IMG_URLS[:2]:
```

Comment:
remove slicing?

Reply:
It makes the tests faster and they all still pass :)

Reply:
haha ok fine w/ me